ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -39,10 +36,10 @@
 
     //banner
     private volatile RelativeLayout adLayout;
-    private VungleNativeAd vungleNativeAd;

```

Comment:
@HuFangshuai need to keep VungleNativeAd for MREC Implementation

Reply:
fixed